Yoast/wordpress-seo
# Only fall back to og:image in Twitter preview if og is enabled

## Context


* Twitter preview showed og:image due to fallback, even when opengraph was disabled. The preview therefore did not match the frontend in this case.

## Summary


This PR can be summarized in the following changelog entry:

* [non-user-facing] Fixes an unreleased bug where Twitter Preview would show the sitewide og:image despite opengraph being disabled, and no og:image being output on the frontend.

## Relevant technical choices:

* Decided to rely on the window object rather than load the setting into redux. Maybe this is not very architecturally correct, because it feels like "polluting" the selectors.

## Test instructions

This PR can be tested by following these steps:

* Test in premium (either `git merge free/P1-70-only-fallback-to-og-image-if-og-is-enabled --no-commit` or copy the relevant code from the `selectors/fallbacks.js` into your premium install. Don't forget to build)
* Set an og image, in Social settings, save, then disable Opengraph, and save.
* Go to a post with no images (no content, no featured image). Basically a new post.
* Verify that the Twitter Preview does not show the sitewide og:image.
* If you want, also test in free, to verify that there are no errors there.

## UI changes

## Documentation

## Quality assurance


Fixes P1-70
